.github/workflows/prepare-for-prod-deploy.yml

```diff
@@ -135,3 +135,20 @@ jobs:
           git push -u origin shared-transformer-$TAG_NAME
 
           hub pull-request -m "chore: upgrade shared transformers to $TAG_NAME"
+
+      - name: Update Helm Chart and Raise Pull Request For Hosted Transformer
+        env:
+          GITHUB_TOKEN: ${{ secrets.PAT }}
+        run: |
+          cd rudder-devops
+          git checkout -b hosted-transformer-$TAG_NAME origin/master
```

Comment:
Rather than hardcoding the default branch name, store it in the previous step and re-use it here.

Reply:
Stored branch name in previous step.